public_html/wp-content/plugins/camptix-attendee-survey/includes/cron.php

```diff
@@ -0,0 +1,183 @@
+<?php
+
+namespace CampTix\AttendeeSurvey\Cron;
+
+defined( 'WPINC' ) || die();
+
+use CampTix_Plugin;
+
+use function CampTix\AttendeeSurvey\Email\{get_email_id, queue_survey_email};
+use function CampTix\AttendeeSurvey\Page\{disable_page, publish_survey_page};
+
+/**
+ * Constants.
+ */
+const DAYS_AFTER_TO_SEND = 2;
+
+/**
+ * Actions & hooks
+ */
+add_action( 'init', __NAMESPACE__ . '\schedule_jobs' );
+add_action( 'wc_attendee_survey_email', __NAMESPACE__ . '\queue_attendee_survey' );
+add_action( 'wc_attendee_disable_survey', __NAMESPACE__ . '\disable_attendee_survey' );
+
+
+/**
+ * Logs a message to the CampTix email log.
+ */
+function log( $message, $post_id, $data = null ) {
+	/* @var CampTix_Plugin $camptix */
+	global $camptix;
+
+	$camptix->log( $message, $post_id, $data );
+}
+
+/**
+ * Add cron jobs to the schedule.
+ */
+function schedule_jobs() {
+	if ( ! wp_next_scheduled( 'wc_attendee_survey_email' ) ) {
+		wp_schedule_event( time(), 'hourly', 'wc_attendee_survey_email' );
```

Comment:
Should it be `daily` instead of `hourly` here?  => Schedule a `daily` cron job that looks if camp ended 2 days ago.

Reply:
Nice catch, `daily` is probably good enough.